frontend/src/components/ViewTrips/trip.js

```diff
@@ -2,8 +2,9 @@ import React, {useState, useEffect} from 'react';
 
 import { Accordion, Button, Card } from 'react-bootstrap';
 
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import authUtils from '../AuthUtils';
-import { timestampToISOString, getDateRangeString } from '../Utils/time.js';
+import { getDateRangeString, timestampToISOString } from "../Utils/time.js";
```

Comment:
Why is this changed?

Reply:
Hmm I don't remember doing that so that must have happened during the crazy merge conflict resolutions before the presentation. Fixed.